staff/oriol-bover/e-commerce-template/index.html

```diff
@@ -0,0 +1,93 @@
+            <input type="text" name="search" class="search_input" placeholder="Search..">
+            <button class="search_button" type="submit"><i class="fas fa-search"></i></button>
+        </form>
+        <div class="menu">
+            <div class="mobile_menu">
+                <i class="fas fa-bars fa-lg"></i>
+            </div>
+            <div class="normal_menu">
+                <ul class="link_list">
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
+            </div>
+        </div>
+    </header>
+    <main class="main">
+        <article class="product">
+            <div class="card">
+                <h2 class="product_header">Product</h2>
+                <img src="http://placehold.it/500x500" class="card-img" />
+                <p class="product_body">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed at efficitur nisl.
+                </p>
+            </div>
+        </article>
+        <article class="product">
+            <div class="card">
+                <h2 class="product_header">Product</h2>
+                <img src="http://placehold.it/500x500" class="card-img" />
+                <p class="product_body">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed at efficitur nisl.
+                </p>
+            </div>
+        </article>
+        <article class="product">
+            <div class="card">
+                <h2 class="product_header">Product</h2>
+                <img src="http://placehold.it/500x500" class="card-img" />
+                <p class="product_body">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed at efficitur nisl.
+                </p>
+            </div>
+        </article>
+        <article class="product">
+            <div class="card">
+                <h2 class="product_header">Product</h2>
+                <img src="http://placehold.it/500x500" class="card-img" />
+                <p class="product_body">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed at efficitur nisl.
+                </p>
+            </div>
+        </article>
+        <article class="product">
+            <div class="card">
+                <h2 class="product_header">Product</h2>
+                <img src="http://placehold.it/500x500" class="card-img" />
+                <p class="product_body">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed at efficitur nisl.
+                </p>
+            </div>
+        </article>
+        <article class="product">
+            <div class="card">
+                <h2 class="product_header">Product</h2>
+                <img src="http://placehold.it/500x500" class="card-img" />
+                <p class="product_body">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed at efficitur nisl.
+                </p>
+            </div>
+        </article>
+
+
+    </main>
+    <footer>
+        Oriol Bover Vila 2019
```

Comment:
this should be inside some tag. like a paragraph.

Reply:
Done